### PR TITLE
Fix several issues with the local docker based cluster documentation.

### PR DESCRIPTION
**Pull request by vishh**
cc @pwittrock

### CODE REVIEW

File: docs/getting-started-guides/docker.md [hunk 1]
```diff
@@ -120,59 +121,69 @@ Now run `docker ps` you should see nginx running.  You may need to wait a few mi
 kubectl expose deployment nginx --port=80
 ```
 
-Run the following command to obtain the IP of this service we just created. There are two IPs, the first one is internal (CLUSTER_IP), and the second one is the external load-balanced IP (if a LoadBalancer is configured)
+Run the following command to obtain the cluster local IP of this service we just created:
 
 ```shell
-kubectl get svc nginx
+ip=$(kubectl get svc nginx --template={{.spec.clusterIP}})
```
Comment (vishh) [inline docs/getting-started-guides/docker.md]: @johndmulhausen @pwittrock: The template doesn't render. How should we fix it?

File: docs/getting-started-guides/docker.md [hunk 2]
```diff
@@ -120,59 +121,69 @@ Now run `docker ps` you should see nginx running.  You may need to wait a few mi
...
+2. Cleanup the filesystem:
+
+On OS X, first ssh into the docker VM:
+
+```shell
+docker-machine ssh `docker-machine active`
+```
+
+```shell
+sudo umount `cat /proc/mounts | grep /var/lib/kubelet | awk '{print $2}'`
```
Comment (pwittrock) [inline docs/getting-started-guides/docker.md]: On my host this fails with:

```
$ docker-machine ssh `docker-machine active` rm -rf /var/lib/kubelet
rm: can't remove '/var/lib/kubelet': Directory not empty
```

and if i use sudo i get

```
docker@default:~$ sudo rm -rf /var/lib/kubelet/
rm: can't remove '/var/lib/kubelet/pods/29e3c012-f2d8-11e5-bbfe-f65e1c279cfc/volumes/kubernetes.io~secret/default-token-3rk30': Device or resource busy
rm: can't remove '/var/lib/kubelet/pods/29e3c012-f2d8-11e5-bbfe-f65e1c279cfc/volumes/kubernetes.io~secret': Directory not empty
rm: can't remove '/var/lib/kubelet/pods/29e3c012-f2d8-11e5-bbfe-f65e1c279cfc/volumes': Directory not empty
rm: can't remove '/var/lib/kubelet/pods/29e3c012-f2d8-11e5-bbfe-f65e1c279cfc': Directory not empty
rm: can't remove '/var/lib/kubelet/pods': Directory not empty
rm: can't remove '/var/lib/kubelet': Directory not empty
```

File: docs/getting-started-guides/docker.md (hunk 1 above)
Comment (johndmulhausen) [inline docs/getting-started-guides/docker.md]: Do this:

126: ```shell{% raw %}
127: ip=$(kubectl get svc nginx --template={{.spec.clusterIP}})
128: echo $ip
129: {% endraw %}```

The issue is the {{ }} matches Liquid syntax

Comment (vishh) [inline docs/getting-started-guides/docker.md]: Awesome. Updated.

**Comment by vishh** [comment]
@pwittrock fixed the templates issue. This should be good to go..

**Comment by pwittrock** [comment]
would you publish the changes if you have not done so already?

**Comment by vishh** [comment]
@pwittrock All changes have been pushed. Github is rendering the diff incorrectly - https://github.com/kubernetes/kubernetes.github.io/pull/250/files#diff-5a22218f3795c88fddf5fdc0ff75453aL131

**Comment by pwittrock** [comment]
Have you pushed to http://vishh.github.io/docs/getting-started-guides/docker/?

**Comment by vishh** [comment]
Updated that now.

**Comment by johndmulhausen** [comment]
Great, thanks guys :)

**Pull request merged**